### DISCUSSION TITLE
Unabl to create `Cluster` resource.

### DISCUSSION THREAD

**Discussion by Magicloud** [cloudnative-pg/cloudnative-pg#238] [Q&A] [answered]
We have a K8S cluster with something configured. I started installing CNPG 1.15.1. The process went well.

Then I started creating an example database like [this](https://cloudnative-pg.io/documentation/1.15.1/quickstart/#part-3-deploy-a-postgresql-cluster) but got error: `Error from server (InternalError): error when creating "db.yaml": Internal error occurred: failed calling webhook "mcluster.kb.io": Post "https://cnpg-webhook-service.cnpg-system.svc:443/mutate-postgresql-cnpg-io-v1-cluster?timeout=10s": Address is not allowed`.

The K8S is in EKS, and I have confirmed that same cluster / database worked with clean EKS. So there must be some conflicts in our K8S. But I could not figure out what it is.

May I have some clues? There is no log in `cnpg-controller-manager` pod about the broken POST request.

**Comment by sxd** [comment, marked as answer]
Hi @Magicloud 

This is a common error when the WebHooks are not up, make sure that the operator shows that your webhook are up and running, and also, make sure that you don't have a security policy blocking the access to the webhooks, and if you must, modify the policies to allow access to the port `443` in the `cnpg-system` namespace to the service `cnpg-webhook-service` and it should work

Regards!

**Reply by Magicloud** [reply]
Silly me, we have weavenet....

**Reply by sxd** [reply]
Great @Magicloud !! hope you can test it soon and report any issue if you found one

Regards!
